src/main/steps/c100-rebuild/check-your-answers/PayAndSubmitPostController.ts

```diff
@@ -0,0 +1,50 @@
+import autobind from 'autobind-decorator';
+import { Response } from 'express';
+
+import { C100_CASE_EVENT } from '../../../app/case/definition';
+import { AppRequest } from '../../../app/controller/AppRequest';
+import { AnyObject, PostController } from '../../../app/controller/PostController';
+import { Form, FormFields, FormFieldsFn } from '../../../app/form/Form';
+import { PaymentHandler } from '../../../modules/payments/paymentController';
+import { C100_CONFIRMATIONPAGE, CHECK_ANSWERS } from '../../../steps/urls';
+
+@autobind
+export default class PayAndSubmitPostController extends PostController<AnyObject> {
+  constructor(protected readonly fields: FormFields | FormFieldsFn) {
+    super(fields);
+  }
+
+  public async post(req: AppRequest<AnyObject>, res: Response): Promise<void> {
+    try {
+      if (req.body.saveAndComeLater) {
+        this.saveAndComeLater(req, res, req.session.userCase);
+      } else {
+        console.log(req.body);
+        const fields = typeof this.fields === 'function' ? this.fields(req.session.userCase) : this.fields;
+        const form = new Form(fields);
+        const { _csrf, ...formData } = form.getParsedBody(req.body);
+        console.log(formData);
+
+        Object.assign(req.session.userCase, formData);
```

Comment:
this can be removed as check your answers will not have any form data which needs to be stored in session

Reply:
Done, removed these lines